### PR TITLE
Make frontend updates in "Create Project" / "Edit Project" flow

### PR DESCRIPTION
**Pull request by DanieleSalatti**
Addresses #299

Previously approved by @codenamejason in  #327, creating new from a branch in this repo instead of using my form as per convo in Discord.

### CODE REVIEW

**Review by codenamejason — APPROVED** [review]
👍

File: client/src/components/base/ProjectForm.tsx [hunk 1]
```diff
@@ -176,9 +179,23 @@ function ProjectForm({
...
-          <p className="text-danger-text w-full text-center font-semibold my-2">
-            {formValidation.message}
-          </p>
+          <div
+            className="p-4 text-red-700 border rounded border-red-900/10 bg-red-50 mt-8"
+            role="alert"
+          >
+            <strong className="text-sm font-medium">
+              There {formValidation.errorCount === 1 ? "was" : "were"}{" "}
+              {formValidation.errorCount} error
```
Comment (gravityblast) [inline client/src/components/base/ProjectForm.tsx]: shall we also put the ending s for plural in case?

Comment (DanieleSalatti) [reply client/src/components/base/ProjectForm.tsx]: Yep, nice catch, I did no pluralise "error(s)"

**Comment by codenamejason** [comment]
Test failing screenshot =>

Screen Shot 2022-09-15 at 5 51 51 PM

**Pull request merged**